packages/styles/forms.css

```diff
@@ -80,6 +80,10 @@ textarea.Field--has-error:focus,
   padding-left: var(--space-half);
 }
 
+.TextFieldContainer {
```

Comment:
`TextFieldContainer` is something specific to the demo/docs app and shouldn't be part of the actual `@deque/cauldron-styles` package.  All you need to do is move this css block to 
https://github.com/dequelabs/cauldron/blob/develop/docs/patterns/components/TextField/index.css

Reply:
Awesome, thanks for the feedback. Moved the css to the correct file in the latest commit